activejob/test/cases/argument_serialization_test.rb

```diff
@@ -49,6 +49,15 @@ class ArgumentSerializationTest < ActiveSupport::TestCase
     assert_arguments_roundtrip([a: 1, "b" => 2])
   end
 
+  test "serialize a ActionController::Parameters" do
+    parameters = ActionController::Parameters.new(a: 1).permit(:a)
```

Comment:
You don't need `ActionController::Parameters` here. You can create a fake class for this test that respond to `permitted?` and `to_h`. This way we simplify the dependencies.

Reply:
Hi, @rafaelfranca

I've followed the folder structure here and add a stub class for the ActiveJob test suite in order to remove the dependency
https://github.com/rails/rails/blob/master/actioncable/test/stubs/global_id.rb

You can take a look, thanks~